safehttp/response_writer_test.go

```diff
@@ -66,6 +66,14 @@ func TestResponseWriterWriteTwicePanic(t *testing.T) {
 				w.Write(safehtml.HTMLEscaped("<h1>Escaped, so not really a heading</h1>"))
 			},
 		},
+		{
+			name: "Call WriteJSON twice",
+			write: func(w *safehttp.ResponseWriter) {
+				obj := struct{ Field string }{Field: "myField"}
+				w.WriteJSON(obj)
+				w.WriteJSON(obj)
+			},
```

Comment:
Add a tests that tries to `w.Write` and then `w.WriteJSON`.

Reply:
I opened an issue for testing every combination and ordering of writing methods. #157